0800-appendix-58067-gramophone.Rmd

```diff
@@ -45,7 +45,10 @@ a maximum of `r fpr::fpr_my_wshd(col = 'elev_max')`m to
 `r fpr::fpr_my_wshd(col = 'elev_site')`m at the crossing (Table
 \@ref(tab:tab-wshd-58067)). Upstream of crossing
 `r as.character(my_site)`, `r fpr::fpr_my_fish_sp()` have previously
-been recorded [@moe2020KnownBC; @norris2020bcfishobs]. The site was revisited in 2023 to better understand fish presence by scoping for barriers, fish sampling to inform prioritization, and to serve as baseline monitoring if the crossing gets fixed. During the 2023 site visit, crews walked from the crossing down to the Bulkley River to assess barriers to fish passage. Approximately 500m upstream from the confluence with the Bulkley River, crews observed a canyon with two falls at ~1.7m and a 25m rock chute. It is presumed that these falls are a barrier to Coho which explains the lack of Coho caught at crossing `r as.character(my_site)` during the 2022 electrofishing survey. Crews also noted that the bottom few hundred meters of stream were de-watered, indicating that timing is an important factor for ocean run fish access to the stream. We believe that some of the fish we caught in 2022 could have been Steelhead due to the historic FISS Steelhead spawning point documented upstream of road, but the falls and de-watering likely explains the lack of coho in 2022. 
+been recorded [@moe2020KnownBC; @norris2020bcfishobs]. 
+
+
+The site was revisited in 2023 to better understand fish presence by scoping for barriers, fish sampling to inform prioritization, and to serve as baseline monitoring if the crossing gets fixed. During the 2023 site visit, crews walked from the crossing down to the Bulkley River to assess barriers to fish passage. Approximately 500m upstream from the confluence with the Bulkley River, crews observed a canyon with two falls at ~1.7m and a 25m rock chute. Coho were captured below the falls, therefore the falls are likely a barrier to Coho which explains the lack of Coho caught at crossing `r as.character(my_site)` upstream during the 2022 electrofishing survey. Crews also noted that the bottom few hundred meters of stream were de-watered, indicating that timing is an important factor for ocean run fish access to the stream. We believe that some of the fish we caught in 2022 could have been Steelhead due to the historic FISS Steelhead spawning point documented upstream of road, but the falls and de-watering likely explains the lack of coho in 2022. 
```

Comment:
The 1.5m high and 1.7m high features are "rock outcrops" (as mentioned in the issue).  We don't call things falls unless they are over 2m high and even then we use that term sparingly (should look like a waterfall in the photos for me to call it a falls).

Description of the obstacles should better describe that the rock outcrops are located at the downstream and upstream end of the chute that is located within a canyon area (like described in the issue)

Coho is inconsistently capitalized in this paragraph.  By convention we don't capitalize species names. (Steelhead should be steelhead too)

Need to tell the reader how we caught the fish.  Dipnetting was conducted below the location of the canyon and rock outcrops.

>Coho caught at crossing `r as.character(my_site)` upstream

coho were captured at electrofishing sites upstream and downstream of Telkwa High Road.

Reply:
Great feedback thanks